signingscript/docker.d/passwords.yml

```
@@ -52,7 +52,8 @@ in:
             - ["https://autograph-external.prod.autograph.services.mozaws.net",
                {"$eval": "AUTOGRAPH_LANGPACK_USERNAME"},
                {"$eval": "AUTOGRAPH_LANGPACK_PASSWORD"},
-               ["autograph_langpack"]
+               ["autograph_langpack"],
+               "cas_cur_webextensions_rsa"
```

I think this should be more like a prod signer:

```suggestion
               "webextensions_rsa_dep_202402"
```

---

Ha wait, no.  That won't work.